src/components/Search/SearchResults/SearchResultItem.module.scss

```diff
@@ -9,33 +9,27 @@
   margin-inline-start: 0;
   margin-inline-end: 0;
   em {
-    color: var(--color-success-medium);
+    font-weight: var(--font-weight-semibold);
```

Comment:
The styling of this file is being used inside `src/components/TarteelVoiceSearch/BodyContainer/SearchResults.tsx` as well. Any change here will affect the styling there.

Reply:
yes, so far seems to produce intended changes.